lib/modules/datasource/sbt-package/index.ts

```diff
@@ -200,6 +193,14 @@ export class SbtPackageDatasource extends MavenDatasource {
       }
     }
 
+    logger.debug(
+      `No versions discovered for ${packageName} listing organization root package folder, fallback to maven datasource for version discovery`
+    );
+    const mavenReleaseResult = await super.getReleases(config);
```

Comment:
Won't this fallback always run, even the original request failed with 404 or any other error?

I don't like to fallback on errors / false positives

Reply:
Wow, missed this point, you're right

Reply:
> Won't this fallback always run, even the original request failed with 404 or any other error?
The fallback is not always run, only when the two list requests fail
eg:
```
https://repo.maven.apache.org/maven2/com/example/
https://repo.maven.apache.org/maven2/com.example/
```
If any of them success the function returns and no fallback is executed

Reply:
@zharinov WDYT?

Reply:
I think it's valid and we can merge this PR